presto-hive/src/test/java/io/prestosql/plugin/hive/parquet/AbstractTestParquetReader.java

```diff
@@ -910,15 +909,13 @@ public void testDecimalBackedByFixedLenByteArray()
 
     @Test
     public void testParquetLongDecimalWriteToPrestoDecimalWithNonMatchingScale()
+            throws Exception
     {
-        assertThatThrownBy(() ->
-                tester.testRoundTrip(
-                        new JavaHiveDecimalObjectInspector(new DecimalTypeInfo(38, 10)),
-                        ImmutableList.of(HiveDecimal.create(0)),
-                        ImmutableList.of(new SqlDecimal(BigInteger.ZERO, 38, 10)),
-                        createDecimalType(38, 9)))
-                .hasMessage("Presto decimal column type has different scale (9) than Parquet decimal column (10)")
```

Comment:
the change here is more that removing `assertThatThrownBy`. Why?

Reply:
Serializing 0 in every scale/precision yields zero so I've changed to values that actually got rescaled